autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -37,16 +62,46 @@ const CourseSelectColumn: React.FC = () => {
         // Card is at least 500px, give it that minimum height
         expandedRowRef.current.className = `${styles.row} ${styles.expandedRow}`;
       }
+
+      // adjust height of section rows
+      if (sectionRows.length > 0) {
+        const col = document.getElementById('course-select-container');
+        if (col) {
+          let otherKidsHeight = 0;
+          for (let i = 0; i < col.children.length; i++) {
+            if (col.children[i] === expandedRowRef.current) {
+            // ignore own height
+            // eslint-disable-next-line no-continue
+              continue;
```

Comment:
```suggestion
              // ignore own height
              // eslint-disable-next-line no-continue
              continue;
```
Aligning comments with block